src/Plugin.php

```diff
@@ -49,9 +49,7 @@ public static function getSubscribedEvents()
 
     public function onPreFileDownload(CPlugin\PreFileDownloadEvent $ev)
     {
-        $url = $ev->getProcessedUrl();
-
-        if (preg_match('/^https?/', $url)) {
```

Comment:
The intent of this code is `$url === 'http:// ....' || $url === 'https:// ....'`.
But it's not working now.